scripts/build.sh

```diff
@@ -139,6 +139,6 @@ function parse_args() {
 	selected_type=$(lower "${selected_type}")
 }
 
-script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+script_dir="$(cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)"
```

Comment:
Let's try it, but I doubt it will solve the problem…

In my experience MSYS2 builds fail randomly ~20% of the time - I need to keep restarting them manually over and over again and need to wait for them to finish before I can merge new PR
(sometime looong time - in worst case I needed to wait 40 minutes I think).

Consequently, I don't like msys2 in general and these 4 jobs are first on the chopping block as soon as we'll need to add new jobs (e.g. 64-bit MSVC build).

Fortunately, [GitHub just added msys2 support to runners](https://github.com/actions/virtual-environments/issues/30) - as soon as preinstalled msys2 will be available (in a week or two), we should replace existing solution and get rid of caches.